### ISSUE TITLE
Update Links for Debian Distro Data

### ISSUE THREAD

**Issue by arshPratap** [openmainframeproject/software-discovery-tool#83]
- **OS** : Ubuntu 20.04

- **Description** : While trying to use the **debian** package data from the PDS repo(as defined [here](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Installation.md#using-data-from-pds)) I ran into the following error : 

  ```
   arsh@arsh-Lenovo-ideapad-320-15IKB:/opt/software-discovery-tool/bin$ sudo ./package_build.py debian 
   Extracting debian data ... 
   Saved!
   filename: Debian_Buster_List.json
   Couldn't pull. Error:  404 File not found
   Thanks for using SDT!
   ```

- **Cause** : I assume the probable cause for this issue is the Debian 11 link that has been provided [here](https://github.com/openmainframeproject/software-discovery-tool/blob/master/bin/package_build.py#L20) seems to be an invalid URL.As directly going onto the link also leads to error 404. 

- **Images** : I have attached some images to validate the above mentioned issue.
         
![Screenshot from 2022-03-15 06-48-20](https://user-images.githubusercontent.com/62841337/158287407-cfb5da53-9f9a-40b1-8246-1acbc96f5ae8.png)
![Screenshot from 2022-03-15 06-48-55](https://user-images.githubusercontent.com/62841337/158287413-d7326b59-026a-46d1-bdfb-7e6d9129930f.png)

- **Possible Solution** : I believe the possible solution would be to change the debian 11 link to debian 11.2 as it seems to work and is currently listed [here](http://ftp.debian.org/debian/)

PS I just recently found about this project and am really interested to contribute to this project and am willing to work on this issue , if I am allowed to work on it by the maintainers

**Comment by pleia2** [comment]
Thanks for the report! Your proposed solution is probably correct since the point releases come quite often, so feel free to test it locally and then submit a pull request with your change for review. Let us know if your have any questions. Welcome to the team! 😄

**Comment by rachejazz** [comment]
Go ahead :)

**Comment by rachejazz** [comment]
@arshPratap btw in the screenshot you attached, you're running the script as root. I'll request you to create another user and give the ownership of SDT to the user and run the script. It's all described in the Installation docs

**Comment by arshPratap** [comment]
Thanks @pleia2 , @rachejazz  for your inputs !! 
I have made a PR regarding this issue. 
Looking forward for your suggestions

**Comment by rachejazz** [comment]
Resolved by #84
